main/lang/english/trad4all.inc.php

```diff
@@ -8067,4 +8067,20 @@
 $ThisEmailWasSentViaCourseXInSessionX = "This e-mail was sent via course %s in session %s";
 $Diagram = "Diagram";
 $CareerXDoesntHaveADiagram = "Career %s doesn't have a diagram.";
+
+// Cloud files in documents strings
+$enable_add_file_link_title = 'Enable adding cloud files links';
+$enable_add_file_link_comment = 'Tool for adding cloud files\' links from external storage sites (Dropbox, Google Drive...) at Documents.';
+$UrlAlreadyExists = "It already exists one equal URL in this folder.";
+$ErrorAddLink = "There was an error adding the link.";
+$AddCloudLink = "Add cloud file link";
+$CloudLinkDeleted = "Cloud file link deleted.";
+$CloudLinkMoved = "Cloud file link moved.";
+$CloudLinkDeleteError = "Error deleting cloud file link.";
+$CloudLinkAdded = "Cloud file link added.";
+$PleaseEnterCloudLinkName = "Please, enter a name for the cloud file link.";
+$MalformedUrl = "URL field format not valid. Expected format example: http://dropbox.com/sh/loremipsum/loremipsum?dl=0";
+$CloudLinkModified = "Cloud file link modified.";
+$NotValidDomain = "Not valid domain. It must be one of these:";
+$ValidDomainsList = "Valid domain list";
```

Comment:
OK, this is a bit complicated but those terms cannot be modified through a PR. You have to send those terms attached to your PR page, so that we can add them to https://translate.chamilo.org, where you can later translate them.

Reply:
Same thing for Spanish and Galician

Reply:
English Strings:
```
$CloudFileLink = "Cloud file link";
$enable_add_file_link_title = 'Enable adding cloud files links';
$enable_add_file_link_comment = 'Tool for adding cloud files\' links from external storage sites (Dropbox, Google Drive...) at Documents.';
$UrlAlreadyExists = "It already exists one equal URL in this folder.";
$ErrorAddLink = "There was an error adding the link.";
$AddCloudLink = "Add cloud file link";
$CloudLinkDeleted = "Cloud file link deleted.";
$CloudLinkMoved = "Cloud file link moved.";
$CloudLinkDeleteError = "Error deleting cloud file link.";
$CloudLinkAdded = "Cloud file link added.";
$PleaseEnterCloudLinkName = "Please, enter a name for the cloud file link.";
$PleaseEnterURL = "Please, enter the URL for the cloud file link.";
$MalformedUrl = "URL field format not valid. Expected format example: http://dropbox.com/sh/loremipsum/loremipsum?dl=0";
$CloudLinkModified = "Cloud file link modified.";
$NotValidDomain = "Not valid domain. It must be one of these:";
$ValidDomainsList = "Valid domain list";
```

Spanish Strings:
```
$CloudFileLink = "Enlace a fichero en la nube";
$enable_add_file_link_title = 'Permitir añadir enlaces a ficheros en la nube';
$enable_add_file_link_comment = 'Herramienta para añadir enlaces a ficheros situados en sitios de almacenamiento externo (Dropbox, Google Drive, etc...) en Documentos.';
$UrlAlreadyExists = "Ya existe una URL igual en este directorio.";
$ErrorAddCloudLink = "Se produjo un error añadiendo el enlace al fichero en la nube.";
$AddCloudLink = "Añadir enlace a un fichero en la nube";
$CloudLinkDeleted = "Enlace a fichero en la nube eliminado.";
$CloudLinkMoved = "Enlace a fichero en la nube movido.";
$CloudLinkDeleteError = "Error al eliminar el enlace al fichero en la nube.";
$CloudLinkAdded = "Enlace a fichero en la nube añadido.";
$PleaseEnterCloudLinkName = "Por favor, introduzca un nombre para el enlace al fichero en la nube.";
$PleaseEnterURL = "Por favor, introduzca la URL del fichero en la nube.";
$MalformedUrl = "Formato del campo URL no válido. Ejemplo del formato esperado: http://dropbox.com/sh/loremipsum/loremipsum?dl=0";
$CloudLinkModified = "Enlace a fichero en la nube modificado.";
$NotValidDomain = "Dominio no válido. Debe de ser uno de los siguientes:";
$ValidDomainsList = "Lista de dominios válidos";
```

Galician Strings:
```
$CloudFileLink = "Enlace a ficheiro na nube";
$enable_add_file_link_title = "Permitir engadir enlaces a ficheiros na nube";
$enable_add_file_link_comment = "Ferramenta para engadir enlaces a ficheiros situados en sitios de almacenamento externo (Dropbox, Google Drive, etc...) en Documentos.";
$UrlAlreadyExists = "Xa existe unha URL igual neste directorio.";
$ErrorAddLink = "Produciuse un erro engadindo o enlace.";
$AddCloudLink = "Engadir enlace a un ficheiro na nube";
$CloudLinkDeleted = "Enlace a ficheiro na nube eliminado.";
$CloudLinkMoved = "Enlace a ficheiro na nube movido.";
$CloudLinkDeleteError = "Erro ao eliminar o enlace ao ficheiro na nube.";
$CloudLinkAdded = "Enlace a ficheiro na nube engadido.";
$PleaseEnterCloudLinkName = "Por favor, introduza un nome para o enlace ao ficheiro na nube.";
$PleaseEnterURL = "Por favor, introduza a URL do ficheiro na nube.";
$MalformedUrl = "Formato do campo URL non válido. Exemplo do formato esperado: http://dropbox.com/sh/loremipsum/loremipsum?dl=0";
$CloudLinkModified = "Enlace a ficheiro na nube modificado.";
$NotValidDomain = "Dominio non válido. Debe ser un dos seguintes:";
$ValidDomainsList = "Lista de dominios válidos";
```